pkg/controller/controller.go

```diff
@@ -43,8 +44,11 @@ func init() {
 }
 
 func getLogger(opts ...zap.Opts) logr.Logger {
+	// make raw logger object from options in original zap pkg
```

Comment:
unnecessary comment. Perhaps rename rawOpts variable to something more fitting like fileLineOpts or defaultOpts.

Reply:
oops deleted the wrong comment - will add the old one.
I'll call em default opts - just because in the future we may need to expand this to include more options, don't wanna limit it to one thing